src/test/java/seedu/address/testutil/TypicalAddressBook.java

```diff
@@ -0,0 +1,25 @@
+package seedu.address.testutil;
```

Comment:
Thank you for moving this to its own class. Could you update your PR to close #69?